src/data_loading.py

```diff
@@ -332,36 +335,82 @@ def get_api_blood_data(api_root = 'https://api.a2cps.org/files/v2/download/publi
 
-    try:
-        # Load Json Data
-        subjects1_filepath = '/'.join([api_root,'subjects','subjects-1-latest.json'])
-        subjects1_request = requests.get(subjects1_filepath)
-        if subjects1_request.status_code == 200:
-            subjects1 = subjects1_request.json()
-        else:
-            return None
-            # return {'status':'500', 'source': api_dict['subjects']['subjects1']}
-
-        subjects2_filepath = '/'.join([api_root,'subjects','subjects-2-latest.json'])
-        subjects2_request = requests.get(subjects2_filepath)
-        if subjects2_request.status_code == 200:
-            subjects2 = subjects2_request.json()
-        else:
-            return None
-            # return {'status':'500', 'source': api_dict['subjects']['subjects2']}
+            subjects2_filepath = '/'.join([api_root,'subjects','subjects-2-latest.json'])
+            subjects2_request = requests.get(subjects2_filepath)
+            if subjects2_request.status_code == 200:
+                subjects2 = subjects2_request.json()
+            else:
+                return None
+                # return {'status':'500', 'source': api_dict['subjects']['subjects2']}
 
-        # Create combined json
-        subjects_json = {'1': subjects1, '2': subjects2}
+            # Create combined json
+            subjects_json = {'1': subjects1, '2': subjects2}
 
-        return subjects_json
+            return subjects_json
 
-    except Exception as e:
-        traceback.print_exc()
+        except Exception as e:
+            traceback.print_exc()
+            return None
+    else:
+        print("Unauthorized attempt to access Subjects data")
         return None
-
+    
+def get_tapis_token(portal_api_root, coresessionid = None):
+    print('get_tapis_token')
+    print(coresessionid)
+    print('portal_api_root')
+    print(portal_api_root)
+    try:
+        response = requests.get(portal_api_root + '/auth/tapis/', headers={'cookie':'coresessionid=' + coresessionid})
+        print('portal api response:')
+        print(response)
+        if response:
+            tapis_token = response.json()['token']
+            return tapis_token
+        else:
+            print("Unauthorized to access tapis token")
+            raise Exception
+    except Exception as e:
+        return('portal api error: {}'.format(e))
+    
+def get_auth_status(vbr_api_root, tapis_token = None):
+    ''' This is the function that will hit the auth check for Life Science API'''
+    print('tapis_token')
+    print(tapis_token)
+    print('vbr_api_root')
+    print(vbr_api_root)
+    try:
+        response = requests.get(vbr_api_root + '/status/auth/', tapis_token)
+        print('vbr api response:')
+        print(response.json()['status'])
+        if response.json()['status'] == 'OK':
+            return True
+        else:
+            print("Unauthorized to access data")
+            raise Exception
```

Comment:
would you want to return False here.  Looks like the user of get_auth_status is expecting this method to return a boolean
```suggestion
            return False
```